logback-classic/src/main/java/ch/qos/logback/classic/encoder/JsonEncoder.java

```diff
@@ -119,6 +122,7 @@ public class JsonEncoder extends EncoderBase<ILoggingEvent> {
     private boolean withMessage = true;
     private boolean withArguments = true;
     private boolean withThrowable = true;
+    private boolean withOriginalStackTrace = false;
```

Comment:
Did you consider `withPlainTextStackTrace` rather than `withOriginalStackTrace` `Original` didn't resonate much with me as the `stepArray is about the same stack trace. I see he difference as a difference of format rather than a difference of data.

Reply:
let's make `withPlainStackTrace`